### ISSUE TITLE
ISTIO 1.1 - ingressgateway not working on a fresh install

### ISSUE THREAD

**Issue by HamzaK8s** [istio/istio#12720]
**Describe the bug**
I installed the new version of istio-1.1 using both option 1 and option 2,  I got the same result, I can't access to the bookinfo through the ingressgateway.

**Expected behavior**
I expected to see logs on the ingressgateway while I'm creating resource like gateway or virtualservice, also I expected to access the bookinfo example through the ingressgateway

**Steps to reproduce the bug**
Installed the istio-1.1 with a fresh install 

```
NAME                                      READY   STATUS    RESTARTS   AGE
istio-citadel-645ffc4999-t6vmf            1/1     Running   0          30m
istio-galley-978f9447f-q7prv              1/1     Running   0          30m
istio-ingressgateway-8ccdc79bc-q69fd      1/1     Running   0          30m
istio-pilot-5f689fc75c-6tnvq              2/2     Running   0          30m
istio-policy-865599cb94-qbg4d             2/2     Running   3          30m
istio-sidecar-injector-6dcc9d5c64-7z4nx   1/1     Running   0          30m
istio-telemetry-d8546d487-ftfnn           2/2     Running   3          30m
prometheus-66c9f5694-zgl5p                1/1     Running   0          30m
```

logs of ingress-gateway
```
2019-03-24T10:30:05.648015Z     info    FLAG: --applicationPorts="[]"
2019-03-24T10:30:05.648084Z     info    FLAG: --binaryPath="/usr/local/bin/envoy"
2019-03-24T10:30:05.648092Z     info    FLAG: --concurrency="0"
2019-03-24T10:30:05.648096Z     info    FLAG: --configPath="/etc/istio/proxy"
2019-03-24T10:30:05.648101Z     info    FLAG: --connectTimeout="10s"
2019-03-24T10:30:05.648105Z     info    FLAG: --controlPlaneAuthPolicy="NONE"
2019-03-24T10:30:05.648162Z     info    FLAG: --controlPlaneBootstrap="true"
2019-03-24T10:30:05.648177Z     info    FLAG: --customConfigFile=""
2019-03-24T10:30:05.648185Z     info    FLAG: --disableInternalTelemetry="false"
2019-03-24T10:30:05.648192Z     info    FLAG: --discoveryAddress="istio-pilot:15010"
2019-03-24T10:30:05.648199Z     info    FLAG: --domain="istio-system.svc.cluster.local"
2019-03-24T10:30:05.648206Z     info    FLAG: --drainDuration="45s"
2019-03-24T10:30:05.648218Z     info    FLAG: --envoyMetricsServiceAddress=""
2019-03-24T10:30:05.648223Z     info    FLAG: --help="false"
2019-03-24T10:30:05.648227Z     info    FLAG: --id=""
2019-03-24T10:30:05.648230Z     info    FLAG: --ip=""
2019-03-24T10:30:05.648234Z     info    FLAG: --lightstepAccessToken=""
2019-03-24T10:30:05.648237Z     info    FLAG: --lightstepAddress=""
2019-03-24T10:30:05.648240Z     info    FLAG: --lightstepCacertPath=""
2019-03-24T10:30:05.648250Z     info    FLAG: --lightstepSecure="false"
2019-03-24T10:30:05.648254Z     info    FLAG: --log_as_json="false"
2019-03-24T10:30:05.648257Z     info    FLAG: --log_caller=""
2019-03-24T10:30:05.648261Z     info    FLAG: --log_output_level="info"
2019-03-24T10:30:05.648264Z     info    FLAG: --log_rotate=""
2019-03-24T10:30:05.648268Z     info    FLAG: --log_rotate_max_age="30"
2019-03-24T10:30:05.648272Z     info    FLAG: --log_rotate_max_backups="1000"
2019-03-24T10:30:05.648285Z     info    FLAG: --log_rotate_max_size="104857600"
2019-03-24T10:30:05.648289Z     info    FLAG: --log_stacktrace_level="default:none"
2019-03-24T10:30:05.648300Z     info    FLAG: --log_target="[stdout]"
2019-03-24T10:30:05.648304Z     info    FLAG: --parentShutdownDuration="1m0s"
2019-03-24T10:30:05.648309Z     info    FLAG: --proxyAdminPort="15000"
2019-03-24T10:30:05.648313Z     info    FLAG: --proxyLogLevel="warning"
2019-03-24T10:30:05.648317Z     info    FLAG: --serviceCluster="istio-ingressgateway"
2019-03-24T10:30:05.648320Z     info    FLAG: --serviceregistry="Kubernetes"
2019-03-24T10:30:05.648324Z     info    FLAG: --statsdUdpAddress=""
2019-03-24T10:30:05.648327Z     info    FLAG: --statusPort="15020"
2019-03-24T10:30:05.648333Z     info    FLAG: --templateFile=""
2019-03-24T10:30:05.648337Z     info    FLAG: --trust-domain=""
2019-03-24T10:30:05.648340Z     info    FLAG: --zipkinAddress="zipkin:9411"
2019-03-24T10:30:05.648356Z     info    Version root@1f4e9557-49cd-11e9-86e9-0a580a2c0304-docker.io/istio-1.1.0-82797c0c0649a3f73029b33957ae105260458c6e-Clean
2019-03-24T10:30:05.648727Z     info    Obtained private IP [172.16.1.179 fe80::7015:6fff:fec8:24a2]
2019-03-24T10:30:05.648850Z     info    Proxy role: &model.Proxy{ClusterID:"", Type:"router", IPAddresses:[]string{"172.16.1.179", "172.16.1.179", "fe80::7015:6fff:fec8:24a2"}, ID:"istio-ingressgateway-8ccdc79bc-q69fd.istio-system", Locality:(*core.Locality)(nil), DNSDomain:"istio-system.svc.cluster.local", ConfigNamespace:"", TrustDomain:"cluster.local", Metadata:map[string]string(nil), SidecarScope:(*model.SidecarScope)(nil), ServiceInstances:[]*model.ServiceInstance(nil)}
2019-03-24T10:30:05.648864Z     info    PilotSAN []string(nil)
2019-03-24T10:30:05.649477Z     info    Effective config: binaryPath: /usr/local/bin/envoy
configPath: /etc/istio/proxy
connectTimeout: 10s
discoveryAddress: istio-pilot:15010
drainDuration: 45s
parentShutdownDuration: 60s
proxyAdminPort: 15000
serviceCluster: istio-ingressgateway
statNameLength: 189
tracing:
  zipkin:
    address: zipkin:9411
2019-03-24T10:30:05.649499Z     info    Monitored certs: []envoy.CertSource{envoy.CertSource{Directory:"/etc/certs/", Files:[]string{"cert-chain.pem", "key.pem", "root-cert.pem"}}}
2019-03-24T10:30:05.649524Z     info    PilotSAN []string(nil)
2019-03-24T10:30:05.649988Z     info    Opening status port 15020
2019-03-24T10:30:05.655883Z     info    Starting proxy agent
2019-03-24T10:30:05.656002Z     info    Received new config, resetting budget
2019-03-24T10:30:05.656014Z     info    Reconciling retry (budget 10)
2019-03-24T10:30:05.656027Z     info    Epoch 0 starting
2019-03-24T10:30:05.657505Z     info    Envoy command: [-c /etc/istio/proxy/envoy-rev0.json --restart-epoch 0 --drain-time-s 45 --parent-shutdown-time-s 60 --service-cluster istio-ingressgateway --service-node router~172.16.1.179~istio-ingressgateway-8ccdc79bc-q69fd.istio-system~istio-system.svc.cluster.local --max-obj-name-len 189 --allow-unknown-fields -l warning]
[2019-03-24 10:30:05.701][13][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.listener.Filter.config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:30:05.701][13][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.Cluster.hosts'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:30:05.701][13][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.Cluster.hosts'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:30:05.701][13][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.Cluster.hosts'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:30:05.701][13][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.config.trace.v2.Tracing.Http.config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:30:05.714][13][warning][config] [bazel-out/k8-opt/bin/external/envoy/source/common/config/_virtual_includes/grpc_stream_lib/common/config/grpc_stream.h:102] gRPC config stream closed: 14, no healthy upstream
[2019-03-24 10:30:05.714][13][warning][config] [bazel-out/k8-opt/bin/external/envoy/source/common/config/_virtual_includes/grpc_stream_lib/common/config/grpc_stream.h:56] Unable to establish new stream
2019-03-24T10:30:08.643346Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:30.642532Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:32.645420Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:34.643814Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:36.644777Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:38.643351Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:40.649722Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:42.643485Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:44.644455Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
[2019-03-24 10:30:44.840][13][warning][config] [bazel-out/k8-opt/bin/external/envoy/source/common/config/_virtual_includes/grpc_stream_lib/common/config/grpc_stream.h:102] gRPC config stream closed: 14, upstream connect error or disconnect/reset before headers
2019-03-24T10:30:46.642383Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:48.643343Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:30:50.643290Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
[2019-03-24 10:31:00.276][13][warning][config] [bazel-out/k8-opt/bin/external/envoy/source/common/config/_virtual_includes/grpc_stream_lib/common/config/grpc_stream.h:102] gRPC config stream closed: 14, upstream connect error or disconnect/reset before headers
2019-03-24T10:31:00.647247Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:02.642839Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:04.650171Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:06.643025Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:08.642493Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:10.653200Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:12.642966Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:14.644475Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:16.644512Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:18.645531Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:20.648423Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:22.645947Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:24.650056Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:26.644971Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 0 successful, 0 rejected; lds updates: 0 successful, 0 rejected
2019-03-24T10:31:28.652839Z     info    Envoy proxy is ready
2019-03-24T10:31:37.795584Z     info    watchFileEvents: "/etc/certs//..2019_03_24_10_31_37.063241460": CREATE
2019-03-24T10:31:37.795609Z     info    watchFileEvents: "/etc/certs//..2019_03_24_10_31_37.063241460": MODIFY|ATTRIB
2019-03-24T10:31:37.795617Z     info    watchFileEvents: "/etc/certs//root-cert.pem": CREATE
2019-03-24T10:31:37.795643Z     info    watchFileEvents: "/etc/certs//cert-chain.pem": CREATE
2019-03-24T10:31:37.795648Z     info    watchFileEvents: "/etc/certs//key.pem": CREATE
2019-03-24T10:31:37.795654Z     info    watchFileEvents: "/etc/certs//..data_tmp": RENAME
2019-03-24T10:31:37.795681Z     info    watchFileEvents: "/etc/certs//..data": CREATE
2019-03-24T10:31:37.795687Z     info    watchFileEvents: "/etc/certs//..2019_03_24_10_30_04.883745671": DELETE
2019-03-24T10:31:47.796826Z     info    watchFileEvents: notifying
2019-03-24T10:31:47.799288Z     info    Received new config, resetting budget
2019-03-24T10:31:47.799425Z     info    Reconciling retry (budget 10)
2019-03-24T10:31:47.799817Z     info    Epoch 1 starting
2019-03-24T10:31:47.806173Z     info    Envoy command: [-c /etc/istio/proxy/envoy-rev1.json --restart-epoch 1 --drain-time-s 45 --parent-shutdown-time-s 60 --service-cluster istio-ingressgateway --service-node router~172.16.1.179~istio-ingressgateway-8ccdc79bc-q69fd.istio-system~istio-system.svc.cluster.local --max-obj-name-len 189 --allow-unknown-fields -l warning]
[2019-03-24 10:31:47.908][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.listener.Filter.config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:31:47.908][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.Cluster.hosts'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:31:47.908][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.Cluster.hosts'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:31:47.908][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.Cluster.hosts'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:31:47.908][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.config.trace.v2.Tracing.Http.config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:31:47.914][13][warning][main] [external/envoy/source/server/server.cc:525] shutting down admin due to child startup
[2019-03-24 10:31:47.914][13][warning][main] [external/envoy/source/server/server.cc:533] terminating parent process
[2019-03-24 10:31:47.922][27][warning][config] [bazel-out/k8-opt/bin/external/envoy/source/common/config/_virtual_includes/grpc_stream_lib/common/config/grpc_stream.h:102] gRPC config stream closed: 14, no healthy upstream
[2019-03-24 10:31:47.922][27][warning][config] [bazel-out/k8-opt/bin/external/envoy/source/common/config/_virtual_includes/grpc_stream_lib/common/config/grpc_stream.h:56] Unable to establish new stream
[2019-03-24 10:33:13.041][13][warning][main] [external/envoy/source/server/server.cc:405] caught SIGTERM
2019-03-24T10:33:13.050777Z     info    Epoch 0 exited normally
[2019-03-24 10:38:51.555][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.listener.Filter.config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:38:51.566][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.config.filter.network.http_connection_manager.v2.HttpFilter.config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:38:51.641][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:38:51.641][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:38:51.641][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:38:51.641][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:45:49.582][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.listener.Filter.config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:45:49.582][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:45:49.582][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:45:49.582][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
[2019-03-24 10:45:49.582][27][warning][misc] [external/envoy/source/common/protobuf/utility.cc:129] Using deprecated option 'envoy.api.v2.route.Route.per_filter_config'. This configuration will be removed from Envoy soon. Please see https://github.com/envoyproxy/envoy/blob/master/DEPRECATED.md for details.
```
The yaml files for gateway and virtualservice 

```
apiVersion: v1
items:
- apiVersion: networking.istio.io/v1alpha3
  kind: Gateway
  metadata:
    creationTimestamp: "2019-03-24T10:38:50Z"
    generation: 1
    name: gw-bookinfo
    namespace: default
    resourceVersion: "142988"
    selfLink: /apis/networking.istio.io/v1alpha3/namespaces/default/gateways/gw-bookinfo
    uid: 02f901a2-4e21-11e9-bc09-000c2953b5e1
  spec:
    selector:
      istio: ingressgateway
    servers:
    - hosts:
      - '*'
      port:
        name: http
        number: 80
        protocol: HTTP
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""

```
```
apiVersion: v1
items:
- apiVersion: networking.istio.io/v1alpha3
  kind: VirtualService
  metadata:
    creationTimestamp: "2019-03-24T10:38:50Z"
    generation: 1
    name: bookinfo-access
    namespace: default
    resourceVersion: "142989"
    selfLink: /apis/networking.istio.io/v1alpha3/namespaces/default/virtualservices/bookinfo-access
    uid: 02fc7837-4e21-11e9-bc09-000c2953b5e1
  spec:
    gateways:
    - gw-bookinfo
    hosts:
    - '*'
    http:
    - match:
      - uri:
          exact: /productpage
      - uri:
          exact: /login
      - uri:
          exact: /logout
      - uri:
          prefix: /api/v1/products
      route:
      - destination:
          host: default/productpage
kind: List
metadata:
  resourceVersion: ""
  selfLink: ""
```

**Version**
```
kubectl version
Client Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.4", GitCommit:"c27b913fddd1a6c480c229191a087698aa92f0b1", GitTreeState:"clean", BuildDate:"2019-02-28T13:37:52Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13", GitVersion:"v1.13.4", GitCommit:"c27b913fddd1a6c480c229191a087698aa92f0b1", GitTreeState:"clean", BuildDate:"2019-02-28T13:30:26Z", GoVersion:"go1.11.5", Compiler:"gc", Platform:"linux/amd64"}
```

**Installation**
I used both option 1 and option 2 

**Environment**
kubernetes

Any help for this issue ?

Thanks

**Comment by hzxuzhonghu** [comment]
This may help you https://istio.io/help/ops/traffic-management/proxy-cmd/#deep-dive-into-envoy-configuration

**Comment by hzxuzhonghu** [comment]
Find this virtualservice setting is not right 
```
      route:
      - destination:
          host: default/productpage
```

**Comment by HamzaK8s** [comment]
@hzxuzhonghu  you right I tought that the namespace scoping was for virtualservice too but when I re-check the release notes 1.1, it was only for gateway
```
Namespace Scoping. When referring to a VirtualService in a Gateway we use DNS based name matching in our configuration model. This can be ambiguous when more than one namespace defines a virtual service for the same host name. To resolve ambiguity it is now possible to explicitly scope these references by namespace using a syntax of the form [{namespace-name}]/{hostname-match} in the hosts field. The equivalent capability is also available in Sidecar for egress.

```
I'll modify it and let you know 
thanks

**Comment by frankbu** [comment]
Fixed by #12804
